zipline/pipeline/filters/filter.py

```diff
@@ -494,3 +496,22 @@ def _compute(self, arrays, dates, assets, mask):
                 asset=self._asset, start_date=dates[0], end_date=dates[-1],
             )
         return out
+
+
+class SpecificAssets(Filter):
+    """
+    A Filter that computes True for a specific set of predetermined assets.
+    """
+    inputs = ()
+    window_length = 0
+    params = ('sids',)
+
+    @expect_types(assets=(list, tuple, np.ndarray))
```

Comment:
how do we get an ndarray, it looks like we will coerce it to a `list` in the coerce below.